src/reporters/JSONReporter.ts

```diff
@@ -13,7 +13,7 @@ export class JSONReporter implements IReporter {
     this.argumentsProvider = argumentsProvider;
   }
 
-  async report(practicesAndComponents: PracticeAndComponent[]): Promise<JSONReport> {
+  report(practicesAndComponents: PracticeAndComponent[]): JSONReport {
```

Comment:
Do we have an information about turned off practices in JSON reporter?

Reply:
Great point. I totally forgot to show, which practices are switched off even in CLI 🙈